diff --git a/dist/src/default-ssdp-options.js b/dist/src/default-ssdp-options.js
index bcce6af79fa2f406a428501b098c3a4b26bb72d4..f687757d3d852eda3229487ca43ba0fd70a2ab78 100644
--- a/dist/src/default-ssdp-options.js
+++ b/dist/src/default-ssdp-options.js
@@ -1,11 +1,8 @@
 import { v4 } from 'uuid';
 import { defaultSocketOptions } from './default-socket-options.js';
 import util from 'util';
-import { createRequire } from 'module';
 import mergeOptions from 'merge-options';
-const req = createRequire(import.meta.url);
-const pkg = req('../../package.json');
-const DEFAULT_SSDP_SIGNATURE = util.format('node.js/%s UPnP/1.1 %s/%s', process.version.substring(1), pkg.name, pkg.version);
+const DEFAULT_SSDP_SIGNATURE = util.format('node.js/%s UPnP/1.1 %s/%s', process.version.substring(1), '@achingbrain/ssdp', '4.0.1');
 export function defaultSsdpOptions(options) {
     return mergeOptions(options ?? {}, {
         usn: `uuid:${v4()}`,
