diff --git a/node_modules/@tamagui/core-node/dist/index.js b/node_modules/@tamagui/core-node/dist/index.js
index 44eb1b3..e7f8caf 100644
--- a/node_modules/@tamagui/core-node/dist/index.js
+++ b/node_modules/@tamagui/core-node/dist/index.js
@@ -6217,10 +6217,6 @@ var require_names = __commonJS({
       white: 4294967295,
       black: 255
     };
-    if (process.env.INCLUDE_CSS_COLOR_NAMES || process.env.TAMAGUI_TARGET === "native") {
-      const allNames = require_names_native().names;
-      Object.assign(names2, allNames);
-    }
   }
 });
 
diff --git a/node_modules/@tamagui/core-node/dist/static.esm.js b/node_modules/@tamagui/core-node/dist/static.esm.js
index d7eee01..ba3571b 100644
--- a/node_modules/@tamagui/core-node/dist/static.esm.js
+++ b/node_modules/@tamagui/core-node/dist/static.esm.js
@@ -6215,10 +6215,6 @@ var require_names = __commonJS({
       white: 4294967295,
       black: 255
     };
-    if (process.env.INCLUDE_CSS_COLOR_NAMES || process.env.TAMAGUI_TARGET === "native") {
-      const allNames = require_names_native().names;
-      Object.assign(names2, allNames);
-    }
   }
 });
 
