diff --git a/node_modules/@tamagui/static/dist/cjs/extractor/bundle.js b/node_modules/@tamagui/static/dist/cjs/extractor/bundle.js
index 0709a69..5138aa5 100644
--- a/node_modules/@tamagui/static/dist/cjs/extractor/bundle.js
+++ b/node_modules/@tamagui/static/dist/cjs/extractor/bundle.js
@@ -87,7 +87,18 @@ function getESBuildConfig({ entryPoints, resolvePlatformSpecificEntries, ...opti
         ...aliases
       })
     ],
-    ...options
+    ...options,
+    resolveExtensions: [
+      '.web.js',
+      '.web.ts',
+      '.web.tsx',
+      '.js',
+      '.jsx',
+      '.json',
+      '.ts',
+      '.tsx',
+      '.mjs',
+    ],
   };
   return res;
 }
